README.md

```diff
@@ -22,35 +22,31 @@ The LSS-ROS2-Arms repository contains common packages that are used by both the
 sudo pip install colcon-common-extensions
 sudo pip install vcstool
 sudo apt install ros-galactic-backward-ros
-sudo apt install python3-rosdep2
 rosdep update --include-eol-distros
 ```
 4. [Ignition Edifice](https://gazebosim.org/docs/edifice/install)
 
 ## Package installation
 
 ```
+cd your-ros-workspace/src
 git clone https://github.com/Lynxmotion/LSS-ROS2-Arms.git
-mkdir -p src
```

Comment:
It was not quite clear to me why this is done. If the LSS-ROS2-Arms repo is closed to `your-workspace/src` then everything works fine and it's tidy.